src/Dao/PlagiabotDao.php

```diff
@@ -48,12 +48,39 @@ public function __construct( $dsn, $user, $pass, $wiki = 'https://en.wikipedia.o
 
 	/**
 	 * @param int $n Number of records asked for
+	 * @param string $filter Filter SQL to show a certian status, one of 'all', 'fixed', 'noaction', 'open' or 'mine'
+	 * @param string $filterUser Filter SQL to only return records reviewed by given user
 	 * @return array|false Data for plagiabot db records or false if no data is not returned
 	 */
-	public function getPlagiarismRecords( $n = 50 ) {
-//		$limit = 'LIMIT ' . $n;
+	public function getPlagiarismRecords( $n = 50, $filter = 'all', $filterUser = NULL ) {
+		$filters = array();
+		$filterSql = '';
+
+		switch ( $filter ) {
+			case 'fixed':
+				array_push( $filters, "status = 'fixed'" );
```

Comment:
$filters[] = "status = 'fixed'"; It is the preferred way when doing MW development. Similarly below.